SkietbaanBE/SkietbaanBE/Properties/launchSettings.json

```diff
@@ -24,7 +24,7 @@
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
       },
-      "applicationUrl": "http://localhost:63475/"
+      "applicationUrl": ""
```

Comment:
what will happen if a person clones the project, installs their nuget packages and tries to run it? Will leaving this blank not cause the run to fail?

Reply:
It will. How should I go about it?

Reply:
Fixed the problem